.eslintrc.yml

```diff
@@ -59,6 +59,12 @@ rules:
   # @typescript-eslint/no-use-before-define is already defined
   no-use-before-define: off
 
+  # Prevent usage of file extension in import statements (except .json and binary files)
```

Comment:
This was added because the current `@typescript-eslint` plugin expects extensions, but we do not want extensions when we import in the project. We need `.json` extensions though because the program will not compile otherwise